bp_common/syn/tcl/vivado_synth.tcl

```diff
@@ -23,18 +24,28 @@ foreach x $f {
       set temp [string trimleft $x $trimchars]
       set expanded [subst $temp]
       lappend dir_list $expanded
+    } elseif {[string match "*bsg_mem_1rw_sync_mask_write_bit.v" $x]} {
```

Comment:
Add a comment

Reply:
# bitmasked memories are incorrectly inferred in <X FPGA>, this version maps into lutram correctly